client/packages/cli/README.md

```diff
@@ -109,15 +109,77 @@ Options:
 - **-t, --timeout \<timeout\>**: Timeout in seconds for waiting for events from the chain. Default timeout is 30 seconds
 - **-x, --export**: Export extrinsic data to a file
 
+### estimate-gas-fee
 
-### estimate
+The `estimate-gas-fee` command is used to estimate the gas fee required for an execution
 
-The `estimate` command is used to estimate the max reward with a given target amount.
+Usage:
+
+```bash
+yarn cli estimate-gas-fee [options]
+```
+
+Options:
+
+- -t, **--target \<name\>**: The target on which is execution will be executed. i.e 'eth', 'roco'
+- -a, **--action \<action\>**: The execution action i.e tass, tran, swap
+- -o, **--args \<speed mode, eth estimation param or SFX JSON string\>**: The execution arguments. It's value can be a speed mode, a EVM call estimation or a side-effect JSON string
+- -s, **--sfx \<file-path\>**: The SFX file path
+- **--signer \<address\>**: The signer's address
+
+Examples:
+```
+# Estimate the gas fee for an asset transfer on ETH target
+p cli estimate-gas-fee -t eth -a tass -o 'fast'
```

Comment:
what is `p cli`?
It should be consistent with usage, without aliases

Reply:
Sorry that's my alias for pnpm.

Fixed; see commit [801c3a8](https://github.com/t3rn/t3rn/pull/1242/commits/801c3a8be8d93b0759177f8d6f8efc28a99a1829).